lang/id/index.md

```diff
@@ -0,0 +1,340 @@
+---
+title: Semantic Versioning 2.0.0
+language: id
+---
+
+Semantic Versioning 2.0.0
+=============================
+
+Ringkasan
+-------
+
+Diberi nomor versi MAJOR.MINOR.PATCH, kenaikan pada:
+
+1. versi MAJOR ketika anda membuat perubahan kompatibilitas API,
+1. versi MINOR ketika anda menambah fungsionalitas pada gaya kompatibilitas sebelumnya dan,
+1. versi PATCH ketika anda membuat perbaikan bug untuk kompatibilitas sebelumnya.
+
+label tambahan untuk pre-release dan build metadata tersedia sebagai ekstensi
+untuk format MAJOR.MINOR.PATCH.
+
+Pengantar
+------------
+
+Dalam dunia manajemen software terdapat tempat menakutkan yang disebut
+"dependency hell" Semakin besar sistem anda tumbuh dan semakin paket anda
+diintegrasikan ke dalam perangkat lunak anda, semakin besar kemungkinan anda
+untuk menemukan diri anda suatu hari pada lingkaran keputusasaan ini.
+
+Pada sistem dengan banyak dependensi, merilis versi paket baru dapat dengan cepat
+menjadi mimpi buruk. Jika spesifikasi dependensi terlalu ketat, anda akan masuk ke
+bahaya dari "version lock" (ketidakmampuan untuk meng-upgrade paket tanpa harus
+merilis versi baru dari setiap paket terkait). Jika dependensi
+ditentukan terlalu longgar, anda pasti akan digigit oleh "version promiscuity"
+(dengan asumsi kompatibilitas dengan versi yang lebih di masa mendatang lebih dari wajar).
+Dependency hell tempat anda berada ketika "version lock" dan/atau "version promiscuity"
+mencegah anda dari kemudahan dan keamanan pergerakan projek anda kedepan.
```

Comment:
Might need to reorder the word to fit Indonesian conversation style, and this is a conclusion but does not include any conclusion word.

Probably can be changed as:

> Dua situasi di atas adalah "dependency hell" yang kita sebutkan, yang membuat kita sulit untuk mengembangkan project lebih lanjut.